src/main/java/teammates/ui/controller/InstructorFeedbackAddAction.java

```diff
@@ -20,15 +16,12 @@
 import teammates.common.exception.TeammatesException;
 import teammates.common.util.Assumption;
 import teammates.common.util.Const;
-import teammates.common.util.EmailType;
 import teammates.common.util.JsonUtils;
 import teammates.common.util.Logger;
-import teammates.common.util.SanitizationHelper;
 import teammates.common.util.StatusMessage;
 import teammates.common.util.StatusMessageColor;
 import teammates.common.util.Templates;
 import teammates.common.util.Templates.FeedbackSessionTemplates;
-import teammates.common.util.TimeHelper;
 import teammates.ui.pagedata.InstructorFeedbacksPageData;
 
 public class InstructorFeedbackAddAction extends InstructorFeedbacksPageAction {
```

Comment:
There are mehtods in `InstructorFeedbacksPageAction` which are used `InstructorFeedbackAddAction`. If we move its code to `InstructorFeedbackAbstractAction` then  `InstructorFeedbackAbstractAction` will have some methods only used in `InstructorFeedbacksPageAction` as other classes extending it may not be using those methods .

So would it be nice to do it like that?